configuration/scripts/options/set_nml.gx3sep2

```diff
@@ -0,0 +1,6 @@
+year_init     = 2005
+month_init    = 9
+day_init      = 2
```

Comment:
Yes it does.  It works just fine.  In fact, sec_init = 7200 as well in this setup in case you didn't notice and it all works fine.  It runs 5 days to 2005-09-07-07200.  The output is also as expected in the sense that things like daily history averages are fine except on the first and last day.  If one started on 2005-09-07-07200, you'd probably want to run 22 hours first (which you can do with npt_unit='h' and npt=22) to get on a daily boundary, and then start running days or months after that.

Reply:
cool, thanks